python/fledge/services/core/interest_registry/change_callback.py

```diff
@@ -43,7 +43,7 @@ async def run(category_name):
     category_value = await cfg_mgr.get_category_all_items(category_name)
     payload = {"category" : category_name, "items" : category_value}
     headers = {'content-type': 'application/json'}
-
+    _LOGGER.error("CHANGE CALLBACK PAYLOAD TO BE SENT %s", json.dumps(payload, sort_keys=True))
```

Comment:
```

Fledge[210332] ERROR: change_callback: fledge.services.core.interest_registry.change_callback: CHANGE CALLBACK PAYLOAD TO BE SENT {"category": "V 1", "items": {"ADHRegions": {"default": "US-West", "description": "AVEVA Data Hub or OSIsoft Cloud Services region", "displayName": "Cloud Service Region", "group": "Cloud", "options": ["US-West", "EU-West", "Australia"], "order": "2", "type": "enumeration", "validity": "PIServerEndpoint == \"AVEVA Data Hub\" || PIServerEndpoint == \"OSIsoft Cloud Services\"", "value": "US-West"}, "AFMap": {"default": "{ }", "description": "Defines a set of rules to address where assets should be placed in the AF hierarchy.", "displayName": "Asset Framework hierarchy rules", "group": "Asset Framework", "order": "17", "type": "JSON", "validity": "PIServerEndpoint == \"PI Web API\"", "value": "{ }"}, "DefaultAFLocation": {"default": "/fledge/data_piwebapi/default", "description": "Defines the default location in the Asset Framework hierarchy in which the assets will be created, each level is separated by /, PI Web API only.", "displayName": "Default Asset Framework Location", "group": "Asset Framework", "order": "16", 

"type": "string", "validity": "PIServerEndpoint == \"PI Web API\"", 
"value": "/fledge\\s/data_piwebapi/default"},

```

Reply:
@pintomax the change callback is good.

Reply:
In the above JSON I cannot see the value. log is truncated.

Please use PIWebAPIUserId which you should see complete.

Set Debug level of the north service.

Then in debug message you'll see the passed string has \s instead of \\s

Reply:
I have already added the value from logs. You might have not seen the comment carefully.